chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/SchedulerManager.java

```diff
@@ -159,6 +162,35 @@ public void deleteTrigger(final String triggerGroup, final String triggerName)
         return triggers;
     }
 
+    @PreAuthorize("@permissions.hasAccess(T(gov.healthit.chpl.permissions.Permissions).SCHEDULER, "
+            + "T(gov.healthit.chpl.permissions.domains.SchedulerDomainPermissions).GET_ALL_SYSTEM_TRIGGERS)")
+    public List<ScheduledSystemJob> getScheduledSystemJobsForUser() throws SchedulerException {
+        List<ScheduledSystemJob> ssJobs = new ArrayList<ScheduledSystemJob>();
+        Scheduler scheduler = getScheduler();
+        for (String group : scheduler.getTriggerGroupNames()) {
+            for (TriggerKey triggerKey : scheduler.getTriggerKeys(groupEquals(group))) {
+                if (scheduler.getTrigger(triggerKey).getJobKey().getGroup().equalsIgnoreCase(SYSTEM_JOBS_KEY)) {
+                    if (doesUserHavePermissionToTrigger(scheduler.getTrigger(triggerKey))) {
```

Comment:
I need to verify that I should be/need to be using this doesUserHavePermissionToTrigger method if check for system jobs (I don't think I do)

Reply:
I removed the check. It does not seem relevant to system jobs and does not cause testing to fail (I re-ran all tests).